setup.py

```diff
@@ -15,7 +15,7 @@
 """Install script for setuptools."""
 
 import os
-from setuptools import find_namespace_packages
+from setuptools import find_packages
```

Comment:
Is this change necessary?

Reply:
I don't think rlax is a namespace so find_packages should suffice. find_namespace_packages add docs to the package which is not desired (I can exclude that of course).